htdocs/compta/facture/class/api_invoices.class.php

```diff
@@ -71,6 +71,25 @@ function get($id)
             throw new RestException(404, 'Invoice not found');
         }
 
+	// Get payment details
+	$this->invoice->totalpaye = $this->invoice->getSommePaiement();
+        $this->invoice->totalcreditnotes = $this->invoice->getSumCreditNotesUsed();
+	$this->invoice->totaldeposits = $this->invoice->getSumDepositsUsed();
+        $this->invoice->resteapayer = price2num($this->invoice->total_ttc - $this->invoice->totalpaye - $this->invoice->totalcreditnotes - $this->invoice->totaldeposits, 'MT');
+
+	// get available discounts
```

Comment:
Its ok for totalpaye, totalcreditnotes, totaldeposits, resteapayer as it is data related to invoice.

However, for data related to customer, this must be moved into api of thirdparties and filled only if module invoice is enabled.